docs/about/history.md

```diff
@@ -0,0 +1,190 @@
+2013, with the work of Salvatore Sanfilippo and Pieter Noordhuis. From May 2013
+to June 2015, Salvatore's work was sponsored by Pivotal.
+
+The Redis Labs era
+------------------
+
+In 2011, a company called Garantia Data was founded and started providing
+database services based on Redis. In 2013, Garantia Data was changing its name
+to RedisDB, but [decided to withdraw the change][redisdb] after complaints by
+Sanfilippo:
+
+[redisdb]: https://techcrunch.com/2014/01/29/database-provider-garantia-data-makes-another-name-change-this-time-to-redis-labs/
+
+> If this is true, it is not a good thing as the current informal rule was: use
+> "Redis" in company names that are selling Redis services, but in a way that
+> makes it distinguishable from Redis as a project. There are many examples like
+> OpenRedis, RedisToGo, and so forth. However "RedisDB" is different as it is
+> more like "We are Redis", (I even own the "redis-db.com" domain name since
+> 2009! you can WHOIS it to check). So in my opinion calling the company
+> "RedisDB" is wrong, especially since I and Garantia Data from time to time
+> have friendly conversations as we are both part of the "Redis ecosystem", but
+> I did not received any prior question about this issue.
+
+The following year, 2014, Garantia Data [changed its name to Redis Labs][redislabs].
+
+[redislabs]: https://techcrunch.com/2014/01/29/database-provider-garantia-data-makes-another-name-change-this-time-to-redis-labs/)
+
+In 2015, Salvatore left Pivotal for Redis Labs. He writes in his blog post
+[Thanks Pivotal, Hello Redis Labs](http://antirez.com/news/91):
+
+> Redis Labs was willing to continue what VMware and Pivotal started. I'll be
+> able to work as I do currently, spending all my time in the open source side
+> of the project, while Redis Labs continues to provide Redis users with an
+> hassles-free Redis experience of managed instances and products.
+
+In 2018, Redis Labs changed the license of some of its modules from AGPL to a
+source-available license. The license prevents competing cloud providers from
+offering these modules to customers and does therefore not fulfil the creteria
+for an open source license.
+
+This was interpreted by some as if Redis is no longer open source. Sanfilippo
+clarified on his blog [Redis will remain BSD licensed][antirez-remain-bsd].
+
+Yiftach Shoolman, CTO and co-founder of Redis Labs, also clarified in in the
+company's blog that [Redis’ License is BSD and will remain
+BSD][will-remain-bsd]. He repeated this promise in [a comment on Hacker
+News][hn-always-bsd], writing “let me assure you that Redis remains and always
+will remain, open source, BSD license”.
+
+[antirez-remain-bsd]: http://antirez.com/news/120
+[will-remain-bsd]: https://redis.io/blog/redis-license-bsd-will-remain-bsd/
+[hn-always-bsd]: https://news.ycombinator.com/item?id=17819392
+
+Also in 2018, Redis Labs aquired the trademark rights to the name Redis and the
```

Comment:
```suggestion
Also in 2018, Redis Labs acquired the trademark rights to the name Redis and the
```

I'm unclear if there is public confirmation of what Redis Labs acquired at the time.

Reply:
Thx. I'll avoid mentioning the year for this.